docs/versions/latest/Release.md

```diff
@@ -81,6 +81,19 @@ If you build project with polyglot + typescript-project-all the build will succe
 
 None
 
+### Installer v 2.27.0 Doesn't Ask for All Required Information about importing vRА
+Installer script doesn't show a prompt to ask for additional information for importing vRA - project name and ID and Organization name and ID
+
+#### Previous Behavior
```

Comment:
Previous and new behavior in the form of:

"It wasn't working"

"now it's working"

doesn't give us any good information as to "why it wasn't working" and "how was it fixed"